### PR TITLE
Rubocop normalization

### PR DESCRIPTION
**Pull request by deivid-rodriguez**
# Description:

Another small step towards normalizing `rubygems` and `bundler` code style.

In this case:

* I enforced no empty lines around class body inside `rubygems` to normalize the code style with the one `bundler` uses.
* ~I enabled `Style/IfInsideElse` inside `rubygems` to normalize the code style with the one `bundler` uses.~ Ended up disabling the cop.

# Tasks:

- [x] Describe the problem / feature
- [ ] Write tests
- [x] Write code to solve the problem
- [ ] Get code review from coworkers / friends

I will abide by the [code of conduct](https://github.com/rubygems/rubygems/blob/master/CODE_OF_CONDUCT.md).

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -109,6 +108,9 @@ Style/Encoding:
   Exclude:
     - test/rubygems/specifications/foo-0.0.1-x86-mswin32.gemspec
 
+Style/IfInsideElse:
```
Comment (deivid-rodriguez) [inline .rubocop.yml:111]: Alternatively, I'm also fine with disabling this rule in `bundler`. I feel this rule sometimes leads to better code, but sometimes it doesn't, so it's probably not worth bothering.

Comment (deivid-rodriguez) [reply .rubocop.yml:111]: I ended up disabling the rule, but I don't have a strong opinion either way.

Comment (hsbt) [reply .rubocop.yml:111]: 👍 I agree to disable it on bundler. `IfInsideElse` is too strict for me.

**Comment by deivid-rodriguez** [comment]
Thanks @hsbt!

**Comment by eregon** [comment]
Thanks for this! :)

**Pull request merged**